packages/eslint-plugin-gestalt/src/eslintASTHelpers.js

```diff
@@ -0,0 +1,233 @@
+// @flow strict
+
+// $FlowFixMe[unclear-type]
+type GenericType = Object;
```

Comment:
@AlbertCarreras `GenericType` is `Object` which is `any`. Can we avoid extracting it and use `any` everywhere? I'm also wondering if we can have types for certain items like `node`.

Reply:
DONE